TinyAlsa: Add support for testing using tinycap and tinyplay. (#1260)
thesofproject/sof-test: README.md

    ...
```

Some tests support SOF_ALSA_TOOL, SOF_ALSA_OPTS, SOF_APLAY_OPTS and SOF_ARECORD_OPTS,
work in progress. Where supported, optional parameters in SOF_APLAY_OPTS
and SOF_ARECORD_OPTS are passed to all aplay and arecord
invocations. SOF_ALSA_OPTS parameters are passed to both aplay and

> arecord. SOF_ALSA_TOOL is used to select tool for testing.

Reviewer: TinyAlsa should also use SOF_(ALSA|APLAY|ARECORD)_OPTS assuming that they follow SOF_ALSA_TOOL value.
Please document this assumption here as well.

Author: Done.

    ...
```

> Some tests support the use of environment variables:
> SOF_ALSA_TOOL, SOF_ALSA_OPTS, SOF_APLAY_OPTS and SOF_ARECORD_OPTS (work in progress)

Reviewer: ```suggestion
Some tests support these environment variables (work in progress):
```

  - SOF_ALSA_TOOL is used to select the audio tool for testing.

>   Set this variable to 'alsa' or 'tinyalsa' to choose between the ALSA and TinyALSA toolsets.

Reviewer: ```suggestion
  Set this variable to 'alsa' (default value) or 'tinyalsa' to choose between the ALSA and TinyALSA toolsets.
```

  - SOF_ALSA_OPTS contains optional parameters passed to both aplay and arecord.
  - SOF_APLAY_OPTS and SOF_ARECORD_OPTS contain optional parameters passed specifically to aplay and arecord.

> These options aply to the selected tool (alsa or tinyalsa) based on the value of SOF_ALSA_TOOL 

Reviewer: ```suggestion
These options are applied to the selected tool (alsa or tinyalsa) based on the value of SOF_ALSA_TOOL 
```

2020-03-19 22:13:32 UTC [COMMAND] aplay -q --fatal-errors  -Dhw:0,0 -r 48000 -c 2 -f S16_LE -d 4 /dev/zero -v -q
    ...
```
Some tests support these environment variables (work in progress):
  - SOF_ALSA_TOOL is used to select the audio tool for testing.
  Set this variable to 'alsa' (default value) or 'tinyalsa' to choose between the ALSA and TinyALSA toolsets.
  - SOF_ALSA_OPTS contains optional parameters passed on both play and record.
  - SOF_APLAY_OPTS and SOF_ARECORD_OPTS contain optional parameters passed additionally on play and record respectively.
These options are applied to the selected tool (alsa or tinyalsa) based on the value of SOF_ALSA_TOOL 

> Warning these environments variables do NOT support parameters

Reviewer 2: ```suggestion
Warning, these environment variables do NOT support parameters
```
